.gitignore

```diff
@@ -25,3 +25,4 @@ Doxyfile-e
 examples/AN01032_100Mbit_avb_i2s_demo/.settings/
 *.pptx
 examples/AN01032_100Mbit_avb_i2s_demo/doc/pdf/
+tests/test_results.csv
```

Comment:
Good rule to add. I think the results file will appear in whichever directory the test script is started from so the rule could simply be `test_results.csv` to catch them all.